libgobuster/dir.go

```diff
@@ -134,13 +134,15 @@ func ProcessDirEntry(s *State, word string, resultChan chan<- Result) {
 		suffix = "/"
 	}
 
-	// Try the DIR first
-	dirResp, dirSize := GoGet(s, s.Url, word+suffix, s.Cookies)
-	if dirResp != nil {
-		resultChan <- Result{
-			Entity: word + suffix,
-			Status: *dirResp,
-			Size:   dirSize,
+	if !(s.ExtensionsOnly) {
```

Comment:
The use of parentheses here isn't necessary.

Reply:
Good point. Fixed in the new file that this is now associated with: gobusterdir.go